src/index.html

```diff
@@ -4,9 +4,354 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo"
+          class="header_img-logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+            apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+             sumsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+            смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#"
+              class="nav__link"
+              data-qa="hover"
```

Comment:
```suggestion
              data-qa="nav-hover"
```
need to be changed according to task conditions